src/System.Runtime.Intrinsics.X86/src/System/Runtime/Intrinsics/X86/Aes.cs

```diff
@@ -0,0 +1,55 @@
+// =+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+//
+// Aes.cs
+//
+// A class that implements intrinsic functions to provide access to Intel AES instructions.
+//
+// =+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+
+using System;
+using System.Runtime.Intrinsics;
+
+namespace System.Runtime.Intrinsics.X86
+{
+    public static class Aes 
+    {
+        public static bool IsSupported { get; }
```

Comment:
I would find it useful to be documented here, at the very least so I don't have to go delving into the CoreCLR repo to find the actual check.

I think it is something some users will care about and may help future contributors find the correct location to add new/missing APIs.